### PR TITLE
Feature/update main menu styles

### PR DESCRIPTION
**Pull request by AdamRatcliffe-TomTom**
Update to main menu screen to match iOS app.

### CODE REVIEW

File: app/src/main/res/drawable/ic_app.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (LukaszWitkowski-tomtom) [inline app/src/main/res/drawable/ic_app.xml:1]: Can you post the screenshot of the old and the new style in the description?

Comment (AdamRatcliffe-TomTom) [reply app/src/main/res/drawable/ic_app.xml:1]: Screenshot_20230518_123254
Screenshot_20230518_123435

Comment (LukaszWitkowski-tomtom) [reply app/src/main/res/drawable/ic_app.xml:1]: For me it looks like the text is moved to the bottom of the icons, is it me or it was moved?

Comment (AdamRatcliffe-TomTom) [reply app/src/main/res/drawable/ic_app.xml:1]: @LukaszWitkowski-tomtom the text isn't as well vertically aligned with the combination of the new icons and font family. I looked into this but as a beginner Android developer I wasn't able to achieve a better alignment. Is this something you could look at?

Comment (LukaszWitkowski-tomtom) [reply app/src/main/res/drawable/ic_app.xml:1]: Done: 
Screenshot_20230522_111330_TomTom SDK Examples

**Review by LukaszWitkowski-tomtom — APPROVED** [review]

**Review by ioanakanda-tomtom — APPROVED** [review]

**Pull request merged**